ui/page/send/send_destination.go

```diff
@@ -167,7 +167,7 @@ func (dst *destination) clearAddressInput() {
 // isSendToAddress returns the current tab selection status without depending
 // on a buffered state.
 func (dst *destination) isSendToAddress() bool {
-	return dst.accountSwitch.SelectedSegment() == values.String(values.StrAddress)
+	return dst.accountSwitch.SelectedSegment() == values.StrAddress
```

Comment:
This is one case where the formal approach caused the send page to not function properly due to an incorrect equality check.

`SelectedSegment()` was returning a pre-translated string(english) and  `values.String(values.StrAddress)` was returning the current string translation(e.g french) after a language change.